synapse/lib/layer.py

```diff
@@ -266,6 +266,10 @@ async def getIden(self):
 
 EDIT_META_SET = 24           # (<etyp>, (<prop>, <valu>, <oldv>, <type>))
 
+# Overwrite edits are converted to normal edits during deconfliction
```

Comment:
A little uncomfortable with EDIT types that don't really exist. Any way we could adjust the implementation to avoid these?

Reply:
We could add a `meta` value back into the nodeedits format and use that in the normal handlers. I'm not a huge fan of that option either though, let me see if I can come up with anything else.